### PR TITLE
[BIOMAGE-748] Add releasing

### PR DESCRIPTION
**Pull request by kafkasl**
Now the nightly build which uses production env will use the latest tag instead of latest commit. This will prevent the tests from failing when we merge new changes that depend on UI code that is not yet released. 

The new paramter ref allows to override that behavior and run whichever commit, branch, or tag on the specified environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ under the repository. You can launch the tests manually for
 each environment.
 
 A nightly test on the production environment only is also run
-at midnight every day automatically.
+at midnight every day automatically. This nightly build uses the latest
+tag on the master branch. If your changes do not depend on
+any repo's release you should create a new tag after merging.
```
Comment (robioson) [inline README.md:44]: `any other`? In fact, maybe rephrase to `If your changes do not depend on any changes in another repo,`

File: .github/workflows/run-e2e.yaml [hunk 2]
```diff
@@ -7,17 +7,42 @@ on:
...
+         Set this variable to override that behavior and run a specific ref regardless of the environment.'
+        required: false
+        default: ''
   schedule:
     - cron:  '22 2 * * *'
 
 jobs:
   run-e2e:
-    name: ${{ format('Run e2e tests for {0}', github.event.inputs.environment || 'production') }}
+    name: ${{ format('Run e2e tests for {0}', github.event.inputs.environment) }}
```
Comment (robioson) [inline .github/workflows/run-e2e.yaml:20]: Why remove this? Was it redundant, i.e. the environment is always set?

**Review by robioson — APPROVED** [review]
Two very minor comments

File: README.md (hunk 1 above)
Comment (kafkasl) [reply README.md]: changed
